codetf.schema.json

```diff
@@ -87,6 +87,34 @@
           "description": "Vendor-specific properties to help storytelling",
           "additionalProperties": true
         },
+        "failureState": {
+          "type": "object",
+          "properties": {
+            "failed": {
```

Comment:
This required `failed` boolean introduces complexity (more opportunities for the data to be invalid) and it makes this a breaking instead of an additive change.

Instead of the boolean, can we let the existence of the failure object indicate a failure? That would make this an additive change and reduce the number of invalid states.

Reply:
Oh I didn't catch that. I thought failureState and this boolean were required

Reply:
In either case, the presence/absence of `failure` seems cleaner.